README.md

```diff
@@ -49,27 +48,28 @@ class Application: Application() {
     override fun onCreate() {
         super.onCreate()
 
-
-        Uppy.init("https://www.example.com/api/", "example-slug", mode)
+        Uppy.init("https://www.example.com/api/", "example-slug")
     }
 }
 
 ```
-* **mode**: an enum of type `UppyMode` used during initialization phase to specify the update modes. There are two possible types of modes:
-  * DEFAULT: This mode is used to show the in-built view elements of the Uppy SDK for Android.
-  * CUSTOM: This mode is used to present customized views and dialogs for the Uppy SDK for Android.
+
 * **slug**: The unique identifier for updates to your application.
 * **cluster**: The unique cluster group within the slug of your application for updates. e.g. "beta", "alpha", "country" etc.
 
 #### Check if the installed app version is up to date:
 
 ```kotlin
-Uppy.checkForUpdates(context, lifecycleOwner)
+    Uppy.checkForUpdates(context, lifecycleOwner)
 ```
-This function should be called at all possible entry points to your application. A common solution is to call it from a base `BaseActivity` or `BaseFragment`.
+This function should be called at all possible entry points to your application. A common solution is to call it from a base `BaseActivity` or `BaseFragment`. This mode is used to show the in-built view elements of the Uppy SDK for Android.
```

Comment:
Why this sentence? 🤔  there is no mode anymore 😂  

You either call the method that checks for updates with a callback or not, depending on the UI you would like to show.

Reply:
You're right!